chat_server/blueprints/personas.py

```diff
@@ -157,4 +163,13 @@ async def toggle_persona_state(
     )
     if updated_data.matched_count == 0:
         raise ItemNotFoundException
+    await _notify_personas_changed()
     return KlatAPIResponse.OK
+
+
+async def _notify_personas_changed():
```

Comment:
I am not quite sure it would be right to have a generic implementation for this notification here as each LLM we use may have dedicated personas set, and emitting only those which are openly available may not give a full picture. I would emit dedicated state update events with persona name, and based on that, it would be for klatchat observer to decide how to forward it appropriately

Reply:
In addition, please consider moving such utility to the file similar to `server_utils/socket_io_utils.py` so we could split view-type logic and utility-type logic

Reply:
> I am not quite sure it would be right to have a generic implementation for this notification here as each LLM we use may have dedicated personas set, and emitting only those which are openly available may not give a full picture. I would emit dedicated state update events with persona name, and based on that, it would be for klatchat observer to decide how to forward it appropriately

That would probably help to minimize changes in `neon-llm-core` too if we scope these per-LLM as the existing queries do. I will make some changes here to try and match the existing responses as closely as possible